modules/jaeger-config-query.adoc

```diff
@@ -0,0 +1,57 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="jaeger-config-query_{context}"]
+= Jaeger Query configuration options
+
+Query is a service that retrieves traces from storage and hosts the user interface to display them.
+
+.Jaeger Query parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+|spec:
+  query:
+   options: {}
+   resources: {}
+|Configuration options that define the Query service.
+|
+|
+
+|query:additional-headers
```

Comment:
These options are not being layered out the same as with the collector and storage sections.

Same as with collector and storage, there are other relevant options defined here: https://www.jaegertracing.io/docs/1.17/cli/#jaeger-query